### PR TITLE
Add getVersion Method

### PR DESCRIPTION
**Pull request by luqz**

### CODE REVIEW

**Comment by kaikai1024** [comment]
@luqz I will check the ci. wait a minute

File: libproto/src/router.rs [hunk 1]
```diff
@@ -89,6 +89,7 @@ pub enum MsgType {
     RequestNewTxBatch,
     RequestNet,
     LocalSync,
+    RequestRpc,
```
Comment (chaoticlonghair) [inline libproto/src/router.rs:92]: Why JSON-RPC call itself has to send a message via channel?

**Review by chaoticlonghair — CHANGES_REQUESTED** [review]

**Comment by kaikai1024** [comment]
could you force-push the commit(just reword the commit) to check the ci status?

File: libproto/src/router.rs (hunk 1 above)
Comment (luqz) [reply libproto/src/router.rs:92]: At the first place can handle both http and websocket calls, the message has been convert to a mq message. If isn't sent via channel, it should be sent to mq and receive from mq.

**Comment by luqz** [comment]
> could you force-push the commit(just reword the commit) to check the ci status?

ok

File: libproto/src/router.rs (hunk 1 above)
Comment (luqz) [reply libproto/src/router.rs:92]: > Why JSON-RPC call itself has to send a message via channel?

Do you think it's better to send it to mq and receive it?

**Pull request closed without merging**